### DISCUSSION TITLE
Add text field type for the multi-line content (Markdown, etc.)

### DISCUSSION THREAD

**Discussion by koddr** [pocketbase/pocketbase#5188] [Q&A] [answered]
Unfortunately, I did not find this question in the roadmap or discussions, so I will ask myself… 😅

I suggest expanding the field types with another text field, in which multi-line text or Markdown can be entered. Now we have the option of using only one-string (`Plain text`) or `Rich editor` fields.

I see two different scenarios here for adding this new type, so as not to break the logic of the Admin UI:

> Add another checkbox for the `Plain text` field settings, for example, `As multi-line text field` or similar, which will apply another component to this field in the UI and DB.

OR

> Add another field type `Text` (and DB type `TEXT`) and rename `Plain text`, for example, to `String` or similar for a better UX.

With any of these options, it is very important that all line breaks `\n` are also stored in the DB.

This will allow you to create even more flexible collections and will allow us to store Markdown markup for later use on the frontend without unnecessary overhead for translating escaped characters of the `Rich editor` field into valid HTML.

@ganigeorgiev I really hope that you will listen and add such an option to your wonderful product! 😉

**Comment by ganigeorgiev** [comment, marked as answer]
The existing `text` field already supports multi-line texts (use `Shift+Enter`) and there are no plans to introduce new text types.

In any case, the existing `editor` field uses TinyMCE and will have to change anyway in the future as the latest TinyMCE v7 is no longer MIT licensed and I'll consider replacing it with an editor that supports Markdown.

**Reply by rebron1900** [reply]
I wonder when this feature will be implemented?
